app/(app)/offence/index.tsx

```diff
@@ -79,13 +94,17 @@ const OffencePage = () => {
             <Field label={t('offence.vehicle')}>
               <TextInput
                 value={ecv}
-                className="font-source-500medium"
+                autoCapitalize="characters"
+                className="font-belfast-700bold text-[18px] color-black"
                 isDisabled={!!role?.actions.scanCheck}
                 onChangeText={(value) => setOffenceState({ ecv: value })}
               />
             </Field>
 
-            <Field label={t('offence.location')}>
+            <Field
+              label={t('offence.location')}
+              errorMessage={isTouched && isLocationError ? t('offence.outOfZone') : undefined}
```

Comment:
What is touched? :D

Reply:
touched means that the form was submitted